src/esp/gfx/Drawable.h

```diff
@@ -54,6 +54,16 @@ class Drawable : public Magnum::SceneGraph::Drawable3D {
 
   Magnum::GL::Mesh& getMesh() { return mesh_; }
 
+  /**
+   * @brief Get the Magnum GL mesh for visualization (e.g., used in object
```

Comment:
Can you explain this a little better? I would say that Drawables are entirely for visualization. That's the whole point of rendering. Do you mean debug visualization specifically?

Reply:
I see. It sounds like "highlight" is the word we want.